services/user.py

```diff
@@ -0,0 +1,52 @@
+from db.models import User
+
+
+def create_user(username,
+                password,
+                email=None,
+                first_name=None,
+                last_name=None):
+    user = User.objects.create(username=username)
```

Comment:
In task we were asked to use create_user method to pass username and password to it at the same time